app/controllers/collaborators_controller.rb

```diff
@@ -58,6 +60,38 @@ def destroy
     end
   end
 
+  # DELETE /collaborators/:id/transfer
```

Comment:
According to `rake routes`, I believe this action is `DELETE /collaborators/:id/destroy_group`

```
destroy_group_collaborator DELETE /collaborators/:id/destroy_group(.:format) collaborators#destroy_group
```